ecvi2.xsd

```diff
@@ -87,14 +88,23 @@
             <xs:attribute name="ExpirationDate" type="xs:date" use="required"/>
             <xs:attribute name="ShipmentDate" type="xs:date" use="optional"/>
             <xs:attribute name="EntryPermitNumber" type="xs:string" use="optional"/>
-            <xs:attribute name="ReplacesCviNumber" type="nonNullString" use="optional"/>
-            <xs:attribute name="Voided" type="xs:boolean" default="false" use="optional">
-                <xs:annotation>
-                    <xs:documentation>True indicates that the CVI named in ReplacesCviNumber has been revoked.
-                                      Voided certificates must include a copy of the human-readable--PDF, etc--
-                                      version of the certificate clearly marked as void.</xs:documentation>
-                </xs:annotation>
-            </xs:attribute>
+            <xs:choice  minOccurs="0" maxOccurs="1">
+                <xs:attribute name="ReplacesCviNumber" type="nonNullString" use="optional">
+                    <xs:annotation>
+                        <xs:documentation>If present, this value is the number of the CVI that is superceded by this
+                                          CVI. The presense of this value also implies that the superceded CVI must
+                                          be considered void, regardless of whether or not there was an explicit
+                                          void request preceding this CVI message.</xs:documentation>
```

Comment:
Not sure if we want any language around it here, but there was some discussion about eCVI software consuming the XML who may want to automatically void a CVI referenced in `ReplacesCviNumber`.